src/Models/HydrostaticFreeSurfaceModels/split_explicit_free_surface_kernels.jl

```diff
@@ -306,7 +306,7 @@ const FNS = FixedSubstepNumber
 const FTS = FixedTimeStepSize
 
 @inline calculate_substeps(substepping::FNS, Δt) = length(substepping.averaging_weights)
-@inline calculate_substeps(substepping::FTS, Δt) = ceil(Int, 2 * Δt / substepping.Δt_barotropic)
+@inline calculate_substeps(substepping::FTS, Δt) = max(5, ceil(Int, 2 * Δt / substepping.Δt_barotropic))
```

Comment:
something like

```suggestion
const MINIMUM_SUBSTEPS = 5
@inline calculate_substeps(substepping::FTS, Δt) = max(MINIMUM_SUBSTEPS, ceil(Int, 2 * Δt / substepping.Δt_barotropic))
```

might be clearer. With a comment around `MINIMUM_SUBSTEPS`! even better